source/renderer/app/config/generateStakePoolFakeData.ts

```diff
@@ -1,33 +1,31 @@
 /**
  * It generates stake pool dummy json content
- * Command to run: node source/renderer/app/config/generateStakePoolFakeData.js
+ * Command to run: npx ts-node source/renderer/app/config/generateStakePoolFakeData.ts
```

Comment:
As we use `yarn` we should also suggest to use it to run this script.
```suggestion
 * Command to run: yarn ts-node source/renderer/app/config/generateStakePoolFakeData.ts
```

Reply:
good catch